src/core/service.c

```diff
@@ -563,6 +563,13 @@ static int service_arm_timer(Service *s, usec_t usec) {
 static int service_verify(Service *s) {
         assert(s);
         assert(UNIT(s)->load_state == UNIT_LOADED);
+        ServiceExecCommand c;
+        ExecCommand *command;
+
+        for (c = 0; c < _SERVICE_EXEC_COMMAND_MAX; c++)
```

Comment:
`for (ServiceExecCommand c = 0; ...`

and move ExecCommand declaration inside the for